docs/source/contribute/op_lowering.md

```diff
@@ -184,3 +184,56 @@ You can find the `OpKind` definition in
 [interned_strings.h](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/core/interned_strings.h).
 If the aten symbol is missing, you can submit a PR like
 [this](https://github.com/pytorch/pytorch/pull/36851).
+
+### Debugging op dispatch
+
+Sometimes even after registering the lowering, PyTorch might not make use
+of it appropriately. We can check what operations the PyTorch dispatcher
+ran by:
+
+1. Build a debug build of PyTorch and PyTorch/XLA
+
+   ```sh
+   # Do this inside the pytorch/xla checkout
+   export DEBUG=1
+   scripts/build_developer.sh
+   ```
+
+2. Run the problematic script or notebook with the
+   `TORCH_SHOW_DISPATCH_TRACE=1` environment variable.
+
+3. Then whenever the PyTorch dispatcher encounters an operation, we'll
+   see a log like this:
+
+   ```
+   [call] op=[aten::einsum], key=[AutogradXLA]
+   ```
+
+   This suggests that the dispatcher wants to call the `aten.einsum`
+   operation with the `AutogradXLA` dispatch key. The best guide level
```

Comment:
Couldn't we say that PyTorch uses the dispatch key to determine which kernel to call and then point to the blog?